velero-plugins/main.go

```diff
@@ -26,6 +27,7 @@ import (
 func main() {
 	veleroplugin.NewServer().
 		RegisterBackupItemAction("openshift.io/01-common-backup-plugin", newCommonBackupPlugin).
+		RegisterBackupItemAction("openshift.io/02-serviceaccount-backup-plugin", newServiceAccountBackupPlugin).
```

Comment:
Minor comment -- I'd move this down a line so the serviceaccount backup and restore plugins are registered next to each other. It doesn't affect functionality, but that's how we've been organizing it when there are both for the same resource type.